main.rb

```diff
@@ -0,0 +1,130 @@
+require_relative './book'
+require_relative './student'
+require_relative './teacher'
+require_relative './classroom'
+require_relative './rental'
+
+class App
+  def initialize
+    puts teacher.id
```

Comment:
- [ ] Please your app is not running from my end, checkout the error message the image below and kindly fix the issue.
![teacher](https://user-images.githubusercontent.com/65852621/146624756-6e0a5c24-e8d5-42e6-b44e-b3274408e698.png)